driver/bpf/fillers.h

```diff
@@ -4019,7 +4019,8 @@ FILLER(sys_getresuid_and_gid_x, true)
 	 * ruid
 	 */
 	idp = (uint32_t *)bpf_syscall_get_argument(data, 0);
-	id = _READ(*idp);
+	if (bpf_probe_read_user(&id, sizeof(idp), (void *)idp))
```

Comment:
what about just using `_READ_USER`? and why do we need `PPM_FAILURE_INVALID_USER_MEMORY`?